qa/workunits/rbd/cli_migration.sh

```diff
@@ -99,104 +129,112 @@ compare_images() {
 }
 
 test_import_native_format() {
-    local base_image=$1
-    local dest_image=$2
+    local base_image_spec=$1
+    local dest_image_spec=$2
 
-    rbd migration prepare --import-only "rbd/${base_image}@2" ${dest_image}
-    rbd migration abort ${dest_image}
+    # if image is from namespace
+    local namespace=""
+    if [[ "$base_image_spec" == rbd/*/* ]]; then
+        namespace=$(basename "$(dirname "$base_image_spec")")
+    fi
+    base_image=$(basename "$base_image_spec")
```

Comment:
I'd move this assignment into the `if` branch for clarity:
```
    # if base image is from namespace
    local base_namespace=""
    local base_image=${base_image_spec}
    if [[ "${base_image_spec}" == rbd/*/* ]]; then
        base_namespace=$(basename "$(dirname "${base_image_spec}")")
        base_image=$(basename "${base_image_spec}")
    fi
```

Reply:
thanks for the Example code, now updated the code as per the suggestion.